CONTRIBUTING.md

```diff
@@ -4,17 +4,42 @@ Hi there! We're thrilled that you'd like to contribute to this project. Your hel
 
 The majority of contributions won't need to touch any Ruby code at all.
 
+## Getting started
+
+Before you can start contributing to Linguist, you'll need to set up your environment first. Clone the repo and run [Bundler](http://gembundler.com/) to install its dependencies.
+
+    git clone https://github.com/github/linguist.git
+    cd linguist/
+    script/bootstrap
```

Comment:
This part reads strangely to me: "Clone the repo and run Bundler" — and then instructions that don't invoke Bundler _directly_ at all.

Reply:
Ah yeah. Will change that.